src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataResource.java

```diff
@@ -185,4 +185,12 @@ ResponseEntity updateCaseDataValue(@PathVariable UUID caseUUID, @PathVariable St
         CaseData caseData = caseDataService.getCaseDataByReference(decodedRef);
         return ResponseEntity.ok(GetCaseResponse.from(caseData, true));
     }
+
+    @Authorised(accessLevel = AccessLevel.READ)
+    @GetMapping(value = "/case/reference/{caseUUID}")
+    ResponseEntity<GetCaseReferenceResponse> getCaseReference(@PathVariable UUID caseUUID) {
+        final String caseRef = caseDataService.getCaseRef(caseUUID);
```

Comment:
This method does not return a reference for deleted cases. I guess we can update it or create a new method. It doesn't look like it is important in any usage before this one.